pkg/instance/constants.go

```diff
@@ -29,7 +29,9 @@ func ProcessingModes() []string {
 	return []string{ProcessingAuto, ProcessingParallel, ProcessingSerial}
 }
 
-//go:embed resource/cbp.exe
+// CbpExecutable is a recompiled binary from code at 'https://ritchielawrence.github.io/cmdow' to avoid false-positive antivirus detection
+//
+//go:embed resource/cbpow.exe
```

Comment:
@maciej-majchrzak-wttech @tomasz-sobczyk-wttech have you been aware of it ?:D 

I am tricking anti-virus to run cmd in the background... I needed to recompile binary to avoid false-positive Nod32 reports